app/src/components/items/overview/external/overview.tsx

```diff
@@ -1,34 +1,13 @@
-import {
-  Box,
-  Flex,
-  Heading,
-  Text,
-  Link,
-  Button,
-  ButtonGroup,
-  TagSet,
-  HorizontalRule,
-} from "@nypl/design-system-react-components";
-import parse from "html-react-parser";
+import { Box, Heading, Link, Text } from "@nypl/design-system-react-components";
 
-const metadataFieldToDisplay = {
-  title: "Title",
-  collection: "Collection",
-  names: "Names",
-  origin: "Date / Origin",
-  tableOfContents: "Table Of Contents",
-  locations: "Library Location",
-  subjects: "Subjects",
-  genres: "Genres",
-  notes: "Notes",
-  physicalDescription: "Physical Description",
-  abstract: "Abstract",
-  languages: "Languages",
-  link: "Link",
-  identifiers: "Identifiers",
-  access: "Access",
-  rights: "Rights",
-  typeOfResource: "Type Of Resource",
+export const parseHtmlString = (html) => {
+  const parser = new DOMParser();
+  const doc = parser.parseFromString(html, "text/html");
+  const anchor = doc.querySelector("a");
+  return {
+    href: anchor?.getAttribute("href") || undefined,
+    text: anchor?.textContent || undefined,
+  };
 };
```

Comment:
Is there any way we could share this function somewhere (I think I noticed a third version of it elsewhere). Doesn't have to be now, could just be a backlog ticket / TODO